README.md

```diff
@@ -54,7 +54,7 @@ integrator = Leapfrog(initial_ϵ)
 #   - multinomial sampling scheme,
 #   - generalised No-U-Turn criteria, and
 #   - windowed adaption for step-size and diagonal mass matrix
-proposal = NUTS{MultinomialTS, GeneralisedNoUTurn}(integrator)
+proposal = NUTS{MultinomialTS, NoUTurn}(integrator)
```

Comment:
Maybe keep the old no-U-turn type names like `GeneralisedNoUTurn` and `StrictGeneralisedNoUTurn` to minimise unnecessary code changes, which might upset some AHMC users. As an alternative, we can create a type alias `NoUTurn = GeneralisedNoUTurn` for convenience.

Reply:
Sure!